vuu-ui/packages/vuu-shell/src/layout-management/SaveLayoutPanel.css

```diff
@@ -37,8 +36,8 @@
 }
 
 .saveLayoutPanel-inputText {
-  color: var(--light-text-secondary, #606477);
-  font-family: Nunito Sans Regular;
+  color: var(--light-text-primary, #15171B);
+  font-family: "Nunito Sans", sans-serif;
```

Comment:
shouldn't ever need font-family, it will be inherited

Reply:
the font family for the input and the header don't match figmas: 
this change vs figma design
![image](https://github.com/ScottLogic/finos-vuu/assets/98337074/da57b040-da37-410c-ba2b-524a073ba252)
![image](https://github.com/ScottLogic/finos-vuu/assets/98337074/61920bf6-ba44-43b2-88b6-0c88c47807f1)

Reply:
> shouldn't ever need font-family, it will be inherited

It looks like salt is overriding the font specified in vuu theme, so if I remove this Arial is rendered:

![image](https://github.com/ScottLogic/finos-vuu/assets/99646608/64f07544-5c4a-4783-b27f-0cd39666dd4c)

Reply:
Fixed in [81fdd42](https://github.com/ScottLogic/finos-vuu/pull/100/commits/81fdd42172bfa22fb5a70589fc0fc3d024e5df62). 

Let me know if the comment for the `Nunito Sans A-Variant` `font-face` is clear enough - also whether the name is appropriate. It's weird that there's not a way of making this variant specifically use the `font-feature-settings` - we're having to specify this explicitly in each stylesheet to get the correct variant of the `font-face`.

Interestingly the `DialogHeader.css` needs to have the `--salt-text-fontFamily` variable modified to apply the correct font, as specifying `font-family` gets overriden:

_With `font-family`_
![image](https://github.com/ScottLogic/finos-vuu/assets/99646608/dc6ed3b8-4bad-4b0e-b36e-8a924a450af9)

_With `--salt-text-fontFamily`_
![image](https://github.com/ScottLogic/finos-vuu/assets/99646608/f64b2a96-78f7-4eb9-8d5d-48ab77e8d1cf)

Whereas `.saveLayoutPanel-inputText` in `SaveLayoutPanel.css` needs to have `font-family` and NOT `--salt-text-fontFamily`, as the latter doesn't do anything:

_With `--salt-text-fontFamily`_
![image](https://github.com/ScottLogic/finos-vuu/assets/99646608/a2921d49-c836-4a17-8b19-78338e810121)

_With `font-family`_
![image](https://github.com/ScottLogic/finos-vuu/assets/99646608/60c61f27-a297-47ad-8316-e52e548f75dd)

Not 100% sure what's going on here. Do you know of any other way where none of this is necessary @heswell?